typescript/src/client/evm/evm.ts

```diff
@@ -213,6 +215,67 @@ export class EvmClient implements EvmClientInterface {
     }
   }
 
+  /**
+   * Exports a CDP EVM account's private key.
+   * It is important to store the private key in a secure place after it's exported.
+   *
+   * @param {ExportServerAccountOptions} options - Parameters for exporting the account.
+   * @param {string} options.address - The address of the account to export.
+   * @param {string} [options.name] - The name of the account to export.
+   * @param {string} [options.idempotencyKey] - An idempotency key.
+   *
+   * @returns A promise that resolves to the exported account's private key,
+   * which is the 32-byte private key hex string without a "0x" prefix.
+   *
+   * @example **With an address**
+   * ```ts
+   * const privateKey = await cdp.evm.exportAccount({
+   *   address: "0x1234567890123456789012345678901234567890",
+   * });
+   * ```
+   *
+   * @example **With a name**
+   * ```ts
+   * const privateKey = await cdp.evm.exportAccount({
+   *   name: "MyAccount",
+   * });
+   * ```
+   */
+  async exportAccount(options: ExportServerAccountOptions): Promise<string> {
+    try {
+      const { publicKey, privateKey } = await generateExportEncryptionKeyPair();
+
+      if (options.address) {
+        const { encryptedPrivateKey } = await CdpOpenApiClient.exportEvmAccount(
+          options.address,
+          {
+            exportEncryptionKey: publicKey,
+          },
+          options.idempotencyKey,
+        );
+        return decryptWithPrivateKey(privateKey, encryptedPrivateKey);
+      }
+
+      if (options.name) {
+        const { encryptedPrivateKey } = await CdpOpenApiClient.exportEvmAccountByName(
+          options.name,
+          {
+            exportEncryptionKey: publicKey,
+          },
+          options.idempotencyKey,
+        );
+        return decryptWithPrivateKey(privateKey, encryptedPrivateKey);
+      }
+
+      throw new Error("Either address or name must be provided");
```

Comment:
nit: might be better to lift this out, because if not then we'll have an error stack trace that looks like:
```
Failed to export account: Either address or name must be provided
```

Which makes it seem like there was an issue exporting the account, when in reality it's a user argument validation issue